src/nspawn/nspawn-settings.c

```diff
@@ -936,7 +936,7 @@ static const char *const user_namespace_ownership_table[_USER_NAMESPACE_OWNERSHI
         [USER_NAMESPACE_OWNERSHIP_AUTO]  = "auto",
 };
 
-DEFINE_STRING_TABLE_LOOKUP(user_namespace_ownership, UserNamespaceOwnership);
+DEFINE_STRING_TABLE_LOOKUP_WITH_BOOLEAN(user_namespace_ownership, UserNamespaceOwnership, USER_NAMESPACE_OWNERSHIP_AUTO);
```

Comment:
This maps `yes` to `auto`, but that's not documented. Is it intentional?

Reply:
Yeah, that makes sense. If taking that way, maybe better to add a short comment in the code?